src/pages/overview/Overview.jsx

```diff
@@ -0,0 +1,44 @@
+import './Overview.css'
+import {Link} from "react-router-dom";
+import {useEffect, useState} from "react";
+import axios from 'axios'
+function Overview() {
+
+ const [all, setAll] = useState([])
+
+    async function fetchPost ()
+    {
+        try {
+            const response = await axios.get('http://localhost:3000/posts')
+            setAll(response.data)
+        } catch (e) {
+            console.error(e)
+            alert('Het laden van de data is mislukt')
+        }
+    }
+
+    useEffect( () => {
+        fetchPost()
+    },[all]);
+
+    return (
+
+        <div className='home-container'>
+            <div><h1>Bekijk alle {all.length} posts op het platform</h1></div>
+
+            {all && all.map((all) => (
```

Comment:
Als `all` altijd een array is (en daar heb je voor gezorgd), dan hoef je geen "undefined check" te doen
```suggestion
            {all.map((all) => (
```